status-im/status-legacy
# Features/move messages status go

This commit does a few things:

1) Move messages to status-go
2) Use message-id computed from status-go
3) Remove old replies

Old message id was used for compatibility of replies with older clients.
Given that v1 is breaking, this is not needed anymore and simplifies
moving messages to status-go. No protocol/data-store change is made, to minimize
changes.

### Testing
- Sending/receiving messages in all kind of chats
- Replies
- Login/logout messages
- Performance of scrolling (just to make sure there are no major regressions) / changing chat
- Delete chats
- Block contacts

status: ready
